.github/workflows/scripts/post_before_script.sh

```diff
@@ -2,9 +2,37 @@
 
 set -euv
 
-sudo sed -i  "s/bindIp: 127.0.0.1/bindIp: 127.0.0.1,$(ip address show dev docker0 | grep -o "inet [0-9]*\.[0-9]*\.[0-9]*\.[0-9]*" | grep -o "[0-9]*\.[0-9]*\.[0-9]*\.[0-9]*")/g" /etc/mongod.conf
+export MONGODB_IP=$(ip address show dev docker0 | grep -o "inet [0-9]*\.[0-9]*\.[0-9]*\.[0-9]*" | grep -o "[0-9]*\.[0-9]*\.[0-9]*\.[0-9]*")
+
+sudo sed -i  "s/bindIp: 127.0.0.1/bindIp: 127.0.0.1,$MONGODB_IP/g" /etc/mongod.conf
 sudo systemctl restart mongod
 
+# update settings to configure our mongo install
+cmd_prefix bash -c "cat >> /etc/pulp/settings.py <<EOF
+PULP2_MONGODB = {
+  'name': 'pulp_database',
+  'seeds': '$MONGODB_IP:27017',
+  'username': 'ci_cd',
+  'password': 'ci_cd',
+  'replica_set': '',
+  'ssl': False,
+  'ssl_keyfile': '',
+  'ssl_certfile': '',
+  'verify_ssl': True,
+  'ca_path': '/etc/pki/tls/certs/ca-bundle.crt',
+}
+EOF"
+
+# Restarting single container services
+cmd_prefix bash -c "s6-svc -r /var/run/s6/services/pulpcore-api"
+cmd_prefix bash -c "s6-svc -r /var/run/s6/services/pulpcore-content"
+cmd_prefix bash -c "s6-svc -d /var/run/s6/services/pulpcore-resource-manager"
+cmd_prefix bash -c "s6-svc -d /var/run/s6/services/pulpcore-worker@1"
+cmd_prefix bash -c "s6-svc -d /var/run/s6/services/pulpcore-worker@2"
+cmd_prefix bash -c "s6-svc -u /var/run/s6/services/new-pulpcore-resource-manager"
+cmd_prefix bash -c "s6-svc -u /var/run/s6/services/new-pulpcore-worker@1"
+cmd_prefix bash -c "s6-svc -u /var/run/s6/services/new-pulpcore-worker@2"
```

Comment:
-d is stopping old tasking system, -u is starting the new tasking system
https://skarnet.org/software/s6/s6-svc.html
it makes sense for the upgrade test, but for this I believe you need -r for the new tasking system